app/uk/gov/hmrc/vatapi/controllers/definition/VatApiDefinition.scala

```diff
@@ -52,65 +52,16 @@ class VatApiDefinition @Inject()(appContext: AppContext) {
             version = "1.0",
             access = buildWhiteListingAccess(),
             status = buildAPIStatus("1.0"),
-            endpoints = allEndpoints,
             endpointsEnabled = true
           )
         ),
         requiresTrust = None
       )
     )
   val logger: Logger = Logger(this.getClass)
-  val vatEndpoints: Seq[Endpoint] = {
-    Seq(
-      Endpoint(
-        uriPattern = "/{vrn}/obligations",
-        endpointName = "Retrieve all VAT obligations",
-        method = GET,
-        authType = USER,
-        throttlingTier = UNLIMITED,
-        scope = Some(readScope),
-        groupName = Vat)
-      ,
-      Endpoint(
-        uriPattern = "/{vrn}/returns",
-        endpointName = "Submit VAT return for period.",
-        method = POST,
-        authType = USER,
-        throttlingTier = UNLIMITED,
```

Comment:
Looks like AuthType, GroupName, HttpMethod and ResourceThrottlingTier are no longer used and can be deleted.